### PR TITLE
allow networkConfig.ingressIPNetworkCIDRs to be configured

### PR DESCRIPTION
**Pull request by madddi**
Make networkConfig.ingressIPNetworkCIDRs configurable via na inventory variable.

### CODE REVIEW

**Comment by abutcher** [comment]
ok to test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_master/templates/master.yaml.v1.j2 [hunk 1]
```diff
@@ -164,6 +164,9 @@ networkConfig:
 # serviceNetworkCIDR must match kubernetesMasterConfig.servicesSubnet
   serviceNetworkCIDR: {{ openshift.common.portal_net }}
   externalIPNetworkCIDRs: {{ openshift_master_external_ip_network_cidrs | default(["0.0.0.0/0"]) | to_padded_yaml(1,2) }}
+{% if openshift_master_ingress_ip_network_cidr %}
```
Comment (abutcher) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Will need to check for defined somehow:

```
{% if openshift_master_ingress_ip_network_cidr is defined %}
```

**Comment by madddi** [comment]
@abutcher Sorry, you're right. I've added the check, could you run the tests again?

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**